packages/@webex/internal-plugin-voicea/src/voicea.ts

```diff
@@ -19,32 +24,41 @@ export class VoiceaChannel extends WebexPlugin implements IVoiceaChannel {
 
   private areCaptionsEnabled: boolean;
 
-  private isTranscribingEnabled:boolean;
+  private isTranscribingEnabled: boolean;
 
   private vmcDeviceId: string;
 
+  private eventProcessor = (e) => {
+    this.seqNum = e.sequenceNumber + 1;
+    switch (e.data.relayType) {
+      case VOICEA_RELAY_TYPES.ANNOUNCEMENT:
+        this.vmcDeviceId = e.headers.from;
+        this.hasVoiceaJoined = true;
+        this.processAnnouncementMessage(e.data.voiceaPayload);
+        break;
+      case VOICEA_RELAY_TYPES.TRANSLATION_RESPONSE:
+        this.processCaptionLanguageResponse(e.data.voiceaPayload);
+        break;
+      case VOICEA_RELAY_TYPES.TRANSCRIPTION:
+        this.processTranscription(e.data.voiceaPayload);
+        break;
+      default:
+        break;
+    }
+  };
   /**
    * Listen to websocket messages
    */
   private listenToEvents() {
-    this.webex.internal.llm.on('event:relay.event', (e) => {
-      this.seqNum = e.sequenceNumber + 1;
-      switch (e.data.relayType) {
-        case VOICEA_RELAY_TYPES.ANNOUNCEMENT:
-          this.vmcDeviceId = e.headers.from;
-          this.hasVoiceaJoined = true;
-          this.processAnnouncementMessage(e.data.voiceaPayload);
-          break;
-        case VOICEA_RELAY_TYPES.TRANSLATION_RESPONSE:
-          this.processCaptionLanguageResponse(e.data.voiceaPayload);
-          break;
-        case VOICEA_RELAY_TYPES.TRANSCRIPTION:
-          this.processTranscription(e.data.voiceaPayload);
-          break;
-        default:
-          break;
-      }
-    });
+    this.webex.internal.llm.on('event:relay.event', this.eventProcessor);
```

Comment:
don't you need to `bind(this)`?

Reply:
ah, yeah I didn't notice that you defined `this.eventProcessor` with an arrow function, so yeah that's why it works.